README.md

```diff
@@ -7,25 +7,60 @@ A [MIRA](https://github.com/TuftsUniversity/MIRA) replacement running on [Hyrax]
 
 ## Development
 
-Get started by cloning the repository and installing the dependencies:
+### dependencies
+* MySQL
+* Redis
+* Path to [FITS](https://projects.iq.harvard.edu/fits/downloads) in the [Hyrax initializer](https://github.com/curationexperts/epigaea/blob/master/config/initializers/hyrax.rb)
+* sidekiq as queue adapter in [application.rb](https://github.com/curationexperts/epigaea/blob/master/config/):        `config.active_job.queue_adapter = :sidekiq`
+
+
+### initial setup
+Get started by cloning the repository, installing the dependencies, and ensuring the test suite runs:
 
 ```sh
 git clone https://github.com/curationexperts/epigaea.git
 cd epigaea
-
-cp .env.sample .env.development
 bundle install
-bundle exec rails db:setup
 bundle exec sidekiq -d -l tmp/sidekiq.log
-# open a separate session and run bundle exec rails hydra:server
+
+# start each of the next services in their own terminal session
+bundle exec rake hydra:server 
+bundle exec rake hydra:test_server
+
+#back in your main terminal session
+bundle exec rails db:setup
+bundle exec rake spec
 ```
-Other services and settings required:
-* MySQL
-* Redis
-* Path to [FITS](https://projects.iq.harvard.edu/fits/downloads) in the [Hyrax initializer](https://github.com/curationexperts/epigaea/blob/master/config/initializers/hyrax.rb)
-* sidekiq as queue adapter in [application.rb](https://github.com/curationexperts/epigaea/blob/master/config/):        `config.active_job.queue_adapter = :sidekiq`
 
-You can run CI with `rake` (or `rake ci`). Or start a server with `rake hydra:server`
+### running the development environment server
+You need to start up the LDAP server using the `ladle` task and Solr, Fedora, and a webserver using the `hydra:server` task.
+It's usually best to run eadh service in it's own terminal session.
```

Comment:
It's usually best to run each service in its own terminal session.

Reply:
Thanks!  Sorry for all the typos - why we shouldn't try to multi-task...